ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -93,7 +66,7 @@ if [[ "$CONFIG_TYPE" == *"externallb"* ]]; then
 fi
 >&2 echo "Created necessary security group rules in ${sg_id}"
 
-server_params=" --image $BASTION_IMAGE --flavor $BASTION_FLAVOR $ZONES_ARGS \
```

Comment:
I refactored the way we configure zones, now done in ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh at the cluster profile level, so we don't need to override it for the jobs.

I figured that it's easier to control on which AZ(s) a cluster profile should be consuming cloud resources, rather than per job, where it's error prone and adds a ton of duplicates.